plugins/forum/lib/foodsoft_forum/engine.rb

```diff
@@ -0,0 +1,20 @@
+module FoodsoftForum
+  class Engine < ::Rails::Engine
+    def navigation(primary, context)
+      return unless FoodsoftForum.enabled?
+      primary.item :forum, I18n.t('navigation.forum'), '/f/forums', id: nil do |subnav|
+        for category in  Forem::Category.all
+          subnav.item :forem_category, category.name, '/f/forums/categories/' + category.name.downcase, id: nil
+        end
+      end
```

Comment:
Hmm I'll try. Perhaps [`default_url_options`](https://github.com/foodcoops/foodsoft/blob/master/app/controllers/application_controller.rb#L186) are not picked up by the engine.